packages/model/src/event.ts

```diff
@@ -37,7 +37,7 @@ export interface EventResponse {
   meetingLink?: string;
 
   calendar: CalendarResponse;
-  groups: GroupResponse[];
+  group?: GroupResponse;
```

Comment:
since this is a contract change i would hope that there is a test that requires modification to account for the change

Reply:
yeah all good